examples/inputs/example_02b/config.yaml

```diff
@@ -11,13 +11,13 @@ base:
 
   learning_config:
     continue_learning: False
-    trained_policies_save_path: learned_strategies/example_02b_base
+    trained_policies_save_path: Null
```

Comment:
you want `null` to set it to "None" - `Null` would be interpreted as string - that path does not exist

Reply:
somehow for me only Null works, not null